.github/workflows/docker_image.yml

```diff
@@ -0,0 +1,127 @@
+name: Docker build and Push
+
+# Trigger the workflow on pull request
+on:
+  pull_request:
+    types: [opened, synchronize, closed]
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    
+    # Checkout the repository to the GitHub Actions runner
+    - uses: actions/checkout@v3
+
+
+    # Set up Python
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
```

Comment:
I reckon we should move to a newer version of Python, say 3.11. Another option is to use a matrix of the most three recent versions of Python. My preference is for the former.

Reply:
I agree.